docs/adding_features.rst

```diff
@@ -39,4 +39,41 @@ to be included in :ref:`GeometricData`, you should:
 After this, you can ``git add`` and ``git commit`` the changes, and make a
 pull request to have them added to the repository.
 
+
+It is not recommended that you modified features directly in
+``geometric_data``, but if you have already done so, you can update the
+manifst of all features and tags based on your changes first, then merge your
+features into a feature colleciton and then split it back out into individual
+features to ensure consistent formatting.
+
+.. code-block:: python
+
+    import os
+    from geometric_features import GeometricFeatures
+    from geometric_features.utils import write_feature_names_and_tags
+
+
+    # Write a file features_and_tags.json with features and tags from the cache.
+    # This updates the file names, feature names and tags that geometric_features
+    # knows about.
+    write_feature_names_and_tags('./geometric_data')
+
+    # move features_and_tags.json into geometric_features to replace the old
+    # manifest
+    os.rename('features_and_tags.json',
+              'geometric_features/features_and_tags.json')
+
+    # Make a geometric features object that gets data from a local cache in
+    # ./geometric_data.  (The remote branch won't matter.)
+    gf = GeometricFeatures(cacheLocation='./geometric_data')
```

Comment:
so this will also make sure that it dos 'not' download data from the main repo if it doesn't find it in the local repo, right?

Reply:
In general, `gf` will still try to download anything you request that it doesn't find in the local cache.  It does this by looking at the manifest, so by updating the manifest to match the local cache first, you ensure that it's not going to try to download anything.